packages/plugins/LoadingIndicator/README.md

```diff
@@ -19,18 +25,38 @@ map.$store.commit('plugin/loadingIndicator/addLoadingKey', key)
 map.$store.commit('plugin/loadingIndicator/removeLoadingKey', key)
 ```
 
-![Loading indicator example](./readme_loadingIndicator_example.png)
-
 The key must be unique and is kept track of via a Set. It can't be added multiple times, and removing it once always removes it altogether. It is advised to use a key like `{my-plugin-or-application-name}-{procedure-name}` to avoid name conflicts. The LoadingIndicator will usually be used for asynchronous code.
 
 As such, **always call `removeLoadingKey` in the `finally` section of your code** to prevent hanging loading indicators.
 
+```js
+// change loader style at runtime
+map.$store.commit('plugin/loadingIndicator/setLoaderStyle', loaderStyle)
+```
+#### Supported `loaderStyle` options
+
+<table align="center">
+  <tr align="center">
+    <td width="33%" align="center"><img src="./assets/VuetifyLoader.PNG" alt="v-progress-linear" height="120px" style="object-fit: none;"><div>v-progress-linear</div></td>
+    <td width="33%" align="center"><img src="./assets/DefaultLoader.PNG" alt="DefaultLoader" height="120px" style="object-fit: contain;"><div>DefaultLoader</div></td>
+    <td width="33%" align="center"><img src="./assets/RingLoader.PNG" alt="RingLoader" height="120px" style="object-fit: contain;"><div>RingLoader</div></td>
+  </tr>
+  <tr align="center">
+    <td width="33%" align="center"><img src="./assets/RollerLoader.PNG" alt="RollerLoader" height="120px" style="object-fit: contain;"><div>RollerLoader</div></td>
+    <td width="33%" align="center"><img src="./assets/CircleLoader.PNG" alt="CircleLoader" height="120px" style="object-fit: contain;"><div>CircleLoader</div></td>
+    <td width="33%" align="center"><img src="./assets/SpinnerLoader.PNG" alt="SpinnerLoader" height="120px" style="object-fit: contain;"><div>SpinnerLoader</div></td>
+  </tr>
+</table>
```

Comment:
The table is currently not properly being rendered when generating the docs.
However, this should be up and running once #158 is merged.

Reply:
Do you want to wait until https://github.com/Dataport/polar/pull/158 is merged?

Reply:
That, or you can merge those changes here to check if it already works.

Reply:
Still not working due to missing assets in the docs folder. I will think about a fix tomorrow.